### ISSUE TITLE
Add a new organization to ROR: 
 
IGEO (Institute of Geosciences, Albania)

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#498]
Summary of request: Add a new organization to ROR

Update record:
Name of organization: 
ROR ID: 
Which part of the record needs to be changed? 
Description of change: 

Merge/split/deprecate records:
Organizations affected by this change:
ROR ID(s) of organization(s) affected by this change: 
How should the record(s) be changed? 

Add record:
Name of organization: IGEO (Institute of Geosciences, Albania)
Website: https://www.geo.edu.al/site
Link to publications: https://www.geo.edu.al/site
Organization type: Archive (An organization involved in stewarding research and cultural heritage materials. Includes libraries, museums, and zoos)
Wikipedia page: 
Wikidata ID: 
ISNI ID: 
GRID ID: 
Crossref Funder ID: 
Aliases: IGJEUM - Instituti i Gjeoshkecave, Ujrave dhe Mjedisit. IGEWE - Institute of Geosciences, Water and Environment)
Labels:
Acronym/abbreviation: IGEO
Related organizations: 
City: Tirana
Country: Albania
Year established: 1968
How will a ROR ID for this organization be used? To identify affiliations for research publications, To identify affiliations for datasets, To be integrated in a scholarly publishing system

Other information about this request:

**Comment by adambuttrick** [comment]
Geoscience research institute in Albania. Publishes monthly seismological and climate-related datasets (https://www.geo.edu.al/newweb/?fq=sizmobuletinet&gj=gj2&kid=36 ; . While technically operating under the Polytechnic University of Tirana, is ssed as an affiliation in peer-reviewed journals (e.g. https://www.geo.edu.al/skedaret/Handy_etal_19_Tectonics.pdf)  and as a primary affiliation (https://www.researchgate.net/profile/Kujtim-Onuzi). Believe this should be added to ROR.

Similar organizations in ROR: https://ror.org/0121whw78 ; https://ror.org/04ge02x20

**Comment by arthurpsmith** [comment]
Yeah, I saw that it was under the university according to the wikipedia page and their about page, but it does seem sufficiently independent (a "national research unit"). Should be in ROR.

**Comment by arthurpsmith** [comment]
I added the ROR for the university under "Related" in the metadata section.

**Comment by adambuttrick** [comment]
Assigned ROR ID https://ror.org/003f21n80 in release v1.1.
